node/src/components/storage.rs

```diff
@@ -649,7 +649,7 @@ impl Storage {
     }
 
     /// Retrieves single switch block by era ID by looking it up in the index and returning it.
-    fn get_switch_block_by_era_id<Tx: Transaction>(
+    pub(crate) fn get_switch_block_by_era_id<Tx: Transaction>(
```

Comment:
Fixed in https://github.com/CasperLabs/casper-node/pull/875/commits/46d2c808cd0716dc0ffa8cf96772ceaca41f9477

Reply:
Looks like `pub(crate)` got left in.  Since we also already have a test-gated impl for Storage, could you move the new `env()` getter there too please?

Reply:
Merged before I saw this last one.  PR to fix it is here: https://github.com/CasperLabs/casper-node/pull/887